models/staged-quote/src/types.ts

```diff
@@ -0,0 +1,42 @@
+import type {
+  BusinessUnit,
+  Cart,
+  Customer,
+  StagedQuote,
+  StagedQuoteDraft,
+  State,
+  QuoteRequest,
+} from '@commercetools/platform-sdk';
+import { TReferenceGraphql } from '@commercetools-test-data/commons';
+import type { TBuilder } from '@commercetools-test-data/core';
+
+// Default
+export type TStagedQuote = Omit<
+  StagedQuote,
+  'businessUnit' | 'customer' | 'quotationCart' | 'quoteRequest' | 'state'
+> & {
+  businessUnit: BusinessUnit;
+  customer: Customer;
+  quotationCart: Cart;
+  quoteRequest: QuoteRequest;
+  state: State;
+};
+
+// Rest
+export type TStagedQuoteRest = StagedQuote;
+export type TStagedQuoteDraft = StagedQuoteDraft;
+
+export type TStagedQuoteGraphql = TStagedQuote & {
+  businessUnitRef: TReferenceGraphql | null;
+  customerRef: TReferenceGraphql | null;
+  quotationCartRef: TReferenceGraphql | null;
+  quoteRequestRef: TReferenceGraphql | null;
```

Comment:
<img width="384" alt="image" src="https://github.com/commercetools/test-data/assets/15024562/0ce09cc6-60b2-4030-8d33-c70e42cd563e">
Nit: `quotationCartRef` and `quoteRequestRef` aren't nullable

Reply:
this is a really good catch, I will go back and review the types in `quoteRequest` because I know that there are some non-nullable fields I did this to there as well, thanks!

Reply:
as it turns out, there were not any non-nullable errors in `QuoteRequest`, but I did find something else to fix, so, that's good I guess...